<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-FL6LD70U" failures="1" tests="12" name="tm.qa.testcases.HomePageTestCase" time="108.036" errors="0" timestamp="26 Feb. 2020 22:12:13 GMT" skipped="0">
  <testcase classname="tm.qa.testcases.HomePageTestCase" name="HomeLink" time="8.745">
    <failure message="expected [true] but found [tm.qa.pages.HomePage@7e928e2f]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [tm.qa.pages.HomePage@7e928e2f]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:442)
at tm.qa.testcases.HomePageTestCase.HomeLink(HomePageTestCase.java:39)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- HomeLink -->
  <testcase classname="tm.qa.testcases.HomePageTestCase" name="signin" time="9.204"/>
  <testcase classname="tm.qa.testcases.HomePageTestCase" name="signup" time="9.818"/>
  <testcase classname="tm.qa.testcases.HomePageTestCase" name="logoTest" time="8.759"/>
  <testcase classname="tm.qa.testcases.HomePageTestCase" name="getAQuote" time="9.578"/>
  <testcase classname="tm.qa.testcases.HomePageTestCase" name="insurancePlan" time="9.772"/>
  <testcase classname="tm.qa.testcases.HomePageTestCase" name="superVisaVsingle" time="9.031"/>
  <testcase classname="tm.qa.testcases.HomePageTestCase" name="bottomgetAQuote" time="8.478"/>
  <testcase classname="tm.qa.testcases.HomePageTestCase" name="AboutUsLink" time="9.314"/>
  <testcase classname="tm.qa.testcases.HomePageTestCase" name="PrivacyAndPolicyLink" time="8.462"/>
  <testcase classname="tm.qa.testcases.HomePageTestCase" name="TermsAndCondiLink" time="8.304"/>
  <testcase classname="tm.qa.testcases.HomePageTestCase" name="EmailsLink" time="8.571"/>
</testsuite> <!-- tm.qa.testcases.HomePageTestCase -->
